src/components/ViewPhotosDialog/index.tsx

```diff
@@ -135,7 +135,7 @@ export default function ViewPhotosDialog(props: ViewPhotosDialogProps): JSX.Elem
               <a href={p.url} target='blank'>
                 <img
                   className='view-photos-dialog-image'
-                  src={`${p.url}?maxHeight=400`}
+                  src={p.url}
```

Comment:
@charlie-tf you probably added the height for a reason, do you remember why?
I removed this because it forces rendering of thumbnails only - if client wants to render a full image we can't do that. We could have the client set the maxHeight if necessary.
Would it break anything here?

Reply:
I don't think it should break anything. Testing in vercel confirms this. This was a copy/paste from terraware-web

Reply:
Awesome, thanks!!